lib/cos.js

```diff
@@ -0,0 +1,134 @@
+
+exports.initialize = function initializeDataSource(dataSource, callback) {
+	//console.log(dataSource.settings);
+	dataSource.connector = new COSConnector(dataSource.settings);
+	process.nextTick(function () {
+		callback && callback();
+	});
+};
+var Connector = require('loopback-connector').Connector;
```

Comment:
Use `const`, not `var`.  See https://loopback.io/doc/en/contrib/style-guide.html#variable-declarations